kbase-extension/static/kbase/js/util/display.js

```diff
@@ -240,11 +240,17 @@ define([
         if (rect.top === 0 && rect.bottom === 0) {
             return false;
         }
+        const notebookStart = document.querySelector('#notebook').getBoundingClientRect().y;
         return (
-            (rect.top >= 0 &&
+            // Top in View
+            (rect.top >= notebookStart &&
                 rect.top <= (window.innerHeight || document.documentElement.clientHeight)) ||
-            (rect.bottom >= 0 &&
-                rect.bottom <= (window.innerHeight || document.documentElement.clientHeight))
+            // Bottom in View
+            (rect.bottom >= notebookStart &&
+                rect.bottom <= (window.innerHeight || document.documentElement.clientHeight)) ||
+            // Top and Bottom span the whole view
+            (rect.top < notebookStart &&
+                rect.bottom > (window.innerHeight || document.documentElement.clientHeight))
         );
     }
```

Comment:
Changed the verticalInViewport util so that it accounts for the header height 📏 📐

Reply:
Also now can handle the situation where a cell is larger than the viewport, and it spans the screen

Reply:
:tada: